pvlib/solarposition.py

```diff
@@ -1340,15 +1341,20 @@ def solar_zenith_analytical(latitude, hourangle, declination):
     )
 
 
-def hour_angle(times, longitude, equation_of_time):
+@renamed_kwarg_warning("0.11.2", "times", "time", "0.12.0")
+def hour_angle(time, longitude, equation_of_time):
     """
     Hour angle in local solar time. Zero at local solar noon.
 
     Parameters
     ----------
-    times : :class:`pandas.DatetimeIndex`
+    time : :class:`pandas.DatetimeIndex`
         Corresponding timestamps, must be localized to the timezone for the
         ``longitude``.
+
+        .. versionchanged:: 0.11.2
+            Renamed from ``times`` to ``time``.
```

Comment:
It would be nice if the docs somehow indicated that `times` is still allowed, for now.  Maybe we should we use `deprecated` instead of `versionchanged` during the deprecation period, and `versionchanged` afterwards?

Reply:
I don't see a point to state that it's still allowed. The less work the better so I vote for version changed only.  Deprecating is already cumbersome and I'm hesitant to touch that require it

Reply:
My line of thought was exactly the same of @AdamRJensen 's. However, I did hesitate about the message. It can be changed to something like:

```rst
.. versionchanged:: 0.12.0
    Renamed from ``times`` to ``time``. ``times`` was deprecated in ``0.11.2``.
```
or
```rst
.. versionchanged:: 0.11.2
    Renamed from ``times`` to ``time``. ``times`` will be removed in ``0.12.0``.
```
which explains both things but clutters the docs IMO.

I don't mind changing it thou.

Reply:
But it's true that in v0.12.0 a follow-up PR to clean up the test that I have pushed now in https://github.com/pvlib/pvlib-python/pull/2237/commits/65e144b200b35d1c1c757158f6d1786c4066ac9b is required